### ISSUE TITLE
DELETE /manifest/{id}

### ISSUE THREAD

**Issue by SaschaAdler** [crkn-rcdr/lapin#8]
This is a temporary call that will be superseded by other, permanent solutions in the future.

`DELETE /manifest/{id}` performs the following:

* deletes `manifest.slug`
* deletes `manifest.public`
* removes the manifest from the item list of all collections that contain it

This will require an update function on `manifest` for deleting `slug` and `public`, a lookup of the `collection/items` view for finding the collections which contain the manifest, and an update function on `collection` for removing the item which can be run in parallel across each collection in question.

It doesn't actually _delete_ the manifest, although I imagine we'll be doing that with all manifests that don't have slugs at some point.

**Comment by RussellMcOrmond** [comment]
Just a reminder that this DELETE should update the `updateInternalmeta` field to ensure that the changes are pushed forward.

https://github.com/crkn-rcdr/Access-Platform/blob/0d02d74a2c6eb31beb78472773ac56b7ff389ec6/Databases/manifest/design/metadatabus/updates/basic.js#L23-L49

**Comment by SaschaAdler** [comment]
Gotcha. I assume this is true for both the manifest and the collections that will no longer contain the manifest.

**Comment by SaschaAdler** [comment]
Assigning this to @DivyaKrishnan26 for the week that I'm gone. See also crkn-rcdr/Access-Platform#59.
